task1/task.py

```diff
@@ -1,3 +1,6 @@
 #Write a function to print even numbers from 1 to 10
 def printEven():
+    for x in [1..10]:
```

Comment:
you should use `range` function, change it.

Reply:
forgot how was `range`

Reply:
very simple, like this `range(1,10)`